fxa-oauth-server/lib/db/memory.js

```diff
@@ -155,8 +155,8 @@ MemoryStore.prototype = {
     client.canGrant = !! client.canGrant;
     client.trusted = !! client.trusted;
     this.clients[hex] = client;
-    client.hashedSecret = client.hashedSecret;
-    client.hashedSecretPrevious = client.hashedSecretPrevious || '';
+    client.hashedSecret = client.hashedSecret ? buf(client.hashedSecret) : null;
+    client.hashedSecretPrevious = client.hashedSecretPrevious ? buf(client.hashedSecretPrevious) : null;
```

Comment:
The memory db was sometimes returning hex strings and sometimes buffers for these fields; this makes them consistently buffers, just like the mysql backend would produce.